Common++/header/MacAddress.h

```diff
@@ -107,16 +106,34 @@ namespace pcpp
 			if(m_IsValid)
 			{
 				#if _MSC_VER >= 1800
-				std::copy(octets.begin(), octets.end(), stdext::checked_array_iterator<uint8_t*>(m_Address, 6));
+				std::copy(octets.begin(), octets.end(), stdext::checked_array_iterator<uint8_t*>(m_Address, sizeof m_Address));
 				#else
 				std::copy(octets.begin(), octets.end(), std::begin(m_Address));
 				#endif
 			}
 
 			return *this;
 		}
+
+		/**
+		 * Returns the size of raw data
+		 * @return The size of raw data
+		 */
+		static constexpr size_t rawDataSize() { return 6; }
```

Comment:
why is it needed to implement the same method twice (one with `constexpr` and one without it)?

Reply:
`constexpr` allows to use this method in expressions calculated on compile time. The simplest example: `std::array<uint8_t, MacAddress::rawDataSize()> raw`.

https://docs.microsoft.com/en-us/cpp/cpp/constexpr-cpp?view=vs-2019

Reply:
Since this method returns a const number (with the value of 6), why is it have to be a method?

Why not creating a const public member of even a macro for that?